sylvia-derive/src/message.rs

```diff
@@ -297,6 +349,39 @@ impl<'a> EnumMessage<'a> {
         let ep_name = msg_ty.emit_ep_name();
         let messages_fn_name = Ident::new(&format!("{}_messages", ep_name), name.span());
 
+        let is_func_empty = match_arms.clone().into_iter().count() == 0;
+
+        let ctx_creation = if !is_func_empty {
+            quote! {let mut ctx = Into::into(ctx);}
+        } else {
+            quote! {}
+        };
+
+        let context_wrap = match msg_ty {
+            MsgType::Query => quote! {},
+            _ => {
+                if !is_func_empty {
+                    quote! {?;ctx.try_into().map_err(Into::into)}
+                } else {
+                    quote! {; Ok(Response::new())}
+                }
+            }
+        };
+        let base_exec = match msg_ty {
```

Comment:
Prints the execution of the base_exec message. This allows for executing some code before the actual message dispatch (used in the ibc_callback context only for now)